src/Queue/BasicTransformer.php

```diff
@@ -49,6 +49,10 @@ public function getSdkByType(string $type)
                 return $this->sdk->articles();
                 break;
 
+            case 'profile':
```

Comment:
fits what I expect, provided the SDK version is recent enough. That is up to the application using this library, there is no `composer.lock` here.